2021-04-06 15:25:51,919 INFO    MainThread:31260 [wandb_setup.py:_flush():69] setting env: {'project': 'genre_classification_prod', 'run_group': 'dev'}
2021-04-06 15:25:51,919 INFO    MainThread:31260 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-06 15:25:51,920 INFO    MainThread:31260 [wandb_init.py:_log_setup():334] Logging user logs to /home/giacomov/udacity/repo/nd0821-c2-build-model-workflow/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/solution/segregate/wandb/run-20210406_152551-thife1kf/logs/debug.log
2021-04-06 15:25:51,920 INFO    MainThread:31260 [wandb_init.py:_log_setup():335] Logging internal logs to /home/giacomov/udacity/repo/nd0821-c2-build-model-workflow/lesson-5-final-pipeline-release-and-deploy/exercises/exercise_14/solution/segregate/wandb/run-20210406_152551-thife1kf/logs/debug-internal.log
2021-04-06 15:25:51,920 INFO    MainThread:31260 [wandb_init.py:init():367] calling init triggers
2021-04-06 15:25:51,921 INFO    MainThread:31260 [wandb_init.py:init():372] wandb.init called with sweep_config: {}
config: {}
2021-04-06 15:25:51,921 INFO    MainThread:31260 [wandb_init.py:init():416] starting backend
2021-04-06 15:25:51,921 INFO    MainThread:31260 [backend.py:_multiprocessing_setup():69] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-06 15:25:51,935 INFO    MainThread:31260 [backend.py:ensure_launched():123] starting backend process...
2021-04-06 15:25:51,942 INFO    MainThread:31260 [backend.py:ensure_launched():127] started backend process with pid: 31297
2021-04-06 15:25:51,944 INFO    MainThread:31260 [wandb_init.py:init():421] backend started and connected
2021-04-06 15:25:51,946 INFO    MainThread:31260 [wandb_init.py:init():461] updated telemetry
2021-04-06 15:25:51,947 INFO    MainThread:31260 [wandb_init.py:init():480] communicating current version
2021-04-06 15:25:52,434 INFO    MainThread:31260 [wandb_init.py:init():485] got version response upgrade_message: "wandb version 0.10.25 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-06 15:25:52,434 INFO    MainThread:31260 [wandb_init.py:init():493] communicating run to backend with 30 second timeout
2021-04-06 15:25:52,671 INFO    MainThread:31260 [wandb_init.py:init():518] starting run threads in backend
2021-04-06 15:25:52,737 INFO    MainThread:31260 [wandb_run.py:_console_start():1476] atexit reg
2021-04-06 15:25:52,738 INFO    MainThread:31260 [wandb_run.py:_redirect():1339] redirect: SettingsConsole.REDIRECT
2021-04-06 15:25:52,738 INFO    MainThread:31260 [wandb_run.py:_redirect():1344] Redirecting console.
2021-04-06 15:25:52,738 INFO    MainThread:31260 [redirect.py:install():213] install start
2021-04-06 15:25:52,739 INFO    MainThread:31260 [redirect.py:install():228] install stop
2021-04-06 15:25:52,739 INFO    MainThread:31260 [redirect.py:install():213] install start
2021-04-06 15:25:52,740 INFO    MainThread:31260 [redirect.py:install():228] install stop
2021-04-06 15:25:52,740 INFO    MainThread:31260 [wandb_run.py:_redirect():1390] Redirects installed.
2021-04-06 15:25:52,740 INFO    MainThread:31260 [wandb_init.py:init():542] run started, returning control to user process
2021-04-06 15:26:07,467 INFO    MainThread:31260 [wandb_run.py:_atexit_cleanup():1446] got exitcode: 0
2021-04-06 15:26:07,468 INFO    MainThread:31260 [wandb_run.py:_restore():1418] restore
2021-04-06 15:26:07,468 INFO    MainThread:31260 [redirect.py:uninstall():232] uninstall start
2021-04-06 15:26:07,468 INFO    MainThread:31260 [redirect.py:_stop():287] _stop: stdout
2021-04-06 15:26:07,468 INFO    MainThread:31260 [redirect.py:_stop():293] _stop closed: stdout
2021-04-06 15:26:07,469 INFO    stdout    :31260 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-04-06 15:26:07,469 INFO    stdout    :31260 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-04-06 15:26:07,469 INFO    MainThread:31260 [redirect.py:_stop():299] _stop joined: stdout
2021-04-06 15:26:07,469 INFO    MainThread:31260 [redirect.py:_stop():301] _stop rd closed: stdout
2021-04-06 15:26:07,469 INFO    MainThread:31260 [redirect.py:uninstall():236] uninstall done
2021-04-06 15:26:07,469 INFO    MainThread:31260 [redirect.py:uninstall():232] uninstall start
2021-04-06 15:26:07,469 INFO    MainThread:31260 [redirect.py:_stop():287] _stop: stderr
2021-04-06 15:26:07,469 INFO    MainThread:31260 [redirect.py:_stop():293] _stop closed: stderr
2021-04-06 15:26:07,470 INFO    stderr    :31260 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-04-06 15:26:07,470 INFO    stderr    :31260 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-04-06 15:26:07,470 INFO    MainThread:31260 [redirect.py:_stop():299] _stop joined: stderr
2021-04-06 15:26:07,470 INFO    MainThread:31260 [redirect.py:_stop():301] _stop rd closed: stderr
2021-04-06 15:26:07,470 INFO    MainThread:31260 [redirect.py:uninstall():236] uninstall done
2021-04-06 15:26:07,648 INFO    MainThread:31260 [wandb_run.py:_wait_for_finish():1568] got exit ret: file_counts {
  wandb_count: 1
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 6708119
  total_bytes: 6708119
}

2021-04-06 15:26:09,650 INFO    MainThread:31260 [wandb_run.py:_wait_for_finish():1568] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  artifact_count: 2
}
pusher_stats {
  uploaded_bytes: 6709258
  total_bytes: 6709258
}

2021-04-06 15:26:10,740 INFO    MainThread:31260 [wandb_run.py:_show_files():1790] logging synced files
